config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: aivenoy/aiven-operator
-  newTag: v0.1.1
+  newName: controller
```

Comment:
Why do we change it here to default  `controller:latest`?

Reply:
its expected to be that way in the CI pipeline, also the change was accidentally introduced in the previous PR. Also i think it was autogenerated that way. I think we should revert it to the previous version and not touch it again.